app/src/main/java/com/hover/stax/SplashScreenForStaxLinkActivity.java

```diff
@@ -0,0 +1,110 @@
+package com.hover.stax;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.work.ExistingPeriodicWorkPolicy;
+import androidx.work.ExistingWorkPolicy;
+import androidx.work.WorkManager;
+
+import com.amplitude.api.Amplitude;
+import com.hover.sdk.api.Hover;
+import com.hover.stax.actions.Action;
+import com.hover.stax.channels.UpdateChannelsWorker;
+import com.hover.stax.database.Constants;
+import com.hover.stax.destruct.SelfDestructActivity;
+import com.hover.stax.home.MainActivity;
+import com.hover.stax.schedules.ScheduleWorker;
+import com.hover.stax.security.BiometricChecker;
+import com.hover.stax.utils.UIHelper;
+import com.hover.stax.utils.Utils;
+
+import static com.hover.stax.database.Constants.AUTH_CHECK;
+
+public class SplashScreenForStaxLinkActivity extends AppCompatActivity implements BiometricChecker.AuthListener {
```

Comment:
I thought so too. The reason why I created that was because:

1. Placing the App link URL intent filter in the same splash screen activity which was the launcher, makes the icon in my device app page disappear and unable to access Stax; even though it runs on USB run build.